smooth-app-bar-layout/src/main/java/me/henrytao/smoothappbarlayout/SmoothAppBarLayout.java

```diff
@@ -151,6 +151,7 @@ public int getCurrentOffset() {
 
   public void setScrollTargetCallback(ScrollTargetCallback scrollTargetCallback) {
     mScrollTargetCallback = scrollTargetCallback;
+    setBehaviorScrollTargetCallback();
```

Comment:
Hi @dammitt 
`setScrollTargetCallback` is used when you want to use another PullDownToRefresh library. In that case, you may want to point to correct `RecyclerView` or `NestedScrollView`. `SwipeRefreshLayout` is one of examples. 

There was an issue opened here https://github.com/henrytao-me/smooth-app-bar-layout/issues/15

Thanks for adding me to as collaborator. I will push the code today and merge your PR. Woot.